torch/testing/_internal/opinfo/core.py

```diff
@@ -896,6 +896,8 @@ class OpInfo:
 
     is_factory_function: bool = False
 
+    is_randomized_result: bool = False
```

Comment:
Can you say more about why we need this flag? The main reason I'm surprised is:

(1) most ops that are involve randomness can be made deterministic by setting various pytorch state, e.g. `torch.mamul_seed` ([doc](https://pytorch.org/docs/stable/generated/torch.manual_seed.html)). I think we can... probably figure out how to reset the state between running the ref/test so the output? (also, we have other rng ops that don't seem to need a special flag in the OpInfo tests today)

(2) inductor does generate randomness eagerly from eager mode, but the tests you're updating aren't actually using inductor (they use AOTDispatcher, aka they are capturing a graph of the same underlying aten ops and replaying them)